### PR TITLE
feat(Visual Stories): Support Visual Stories

### PR DESCRIPTION
**Pull request by shreyaa1**

### CODE REVIEW

**Comment by iamgarima** [comment]
Support for

- [ ] Seo
- [ ] Caching

File: README.md [hunk 1]
```diff
@@ -168,6 +168,21 @@ The handler can return the following:
...
 * A url starting with http(s) - The URL will be fetched and content will be returned according to the above two rules
 
+## Visual Stories (amp stories)
+
+In order to use the visual-stories, do the following in `app/server/app.js`
+
+```javascript
+import {enableVisualStories} from '@quintype/framework/server/visual-stories';
+
+function renderVisualStory({story, config, client}) {
```
Comment (shreyaa1) [inline README.md]: fix docs (res, story, {config, client})

File: package.json [hunk 2]
```diff
@@ -42,6 +42,7 @@
     "react-redux": "^5.0.6",
     "react-router": "^4.2.0",
     "redux": "^3.7.2",
+    "request": "^2.88.0",
```
Comment (shreyaa1) [inline package.json]: remove this

File: server/handlers/isomorphic-handler.js [hunk 3]
```diff
@@ -94,6 +97,16 @@ function createStoreFromResult(url, result, opts = {}) {
   return createBasicStore(result, qt, opts);
 }
 
+exports.handleVisualStoryRoute = function handleVisualStoryRoute(req, res, next, {config, client, renderVisualStory, loadData, pickComponent, loadErrorData, seo, logError, assetHelper}) {
```
Comment (shreyaa1) [inline server/handlers/isomorphic-handler.js]: delete this

**Pull request merged**